### PR TITLE
14x Ubuntu 22.04 PHP 8.3 Default

### PR DESCRIPTION
**Pull request by svpernova09**
Ubuntu 22.04
PHP 8.3 Default
Add Golang to base box
Remove Drush Launcher
Remove Drupal Launcher
Replace MailHog with Mailpit

### CODE REVIEW

**Comment by 1RV34** [comment]
I'm not sure but shouldn't the line
```php
touch /home/vagrant/.homestead-features/php8.3
```
instead be
```php
touch /home/vagrant/.homestead-features/php83
```
?

**Comment by 1RV34** [comment]
I'm not sure but shouldn't the line
```php
touch /home/vagrant/.homestead-features/php8.3
```
instead be
```php
touch /home/vagrant/.homestead-features/php83
```
?
The `touch`es for the previous PHP versions were without `.`.

File: scripts/amd64.sh [hunk 1]
```diff
@@ -473,6 +474,51 @@ else
...
+  printf "curl.cainfo = /etc/ssl/certs/ca-certificates.crt\n" | tee -a /etc/php/8.3/fpm/php.ini
+
+  # Configure FPM
+  sed -i "s/user = www-data/user = vagrant/" /etc/php/8.3/fpm/pool.d/www.conf
+  sed -i "s/group = www-data/group = vagrant/" /etc/php/8.3/fpm/pool.d/www.conf
+  sed -i "s/listen\.owner.*/listen.owner = vagrant/" /etc/php/8.3/fpm/pool.d/www.conf
+  sed -i "s/listen\.group.*/listen.group = vagrant/" /etc/php/8.3/fpm/pool.d/www.conf
+  sed -i "s/;listen\.mode.*/listen.mode = 0666/" /etc/php/8.3/fpm/pool.d/www.conf
+
+  touch /home/vagrant/.homestead-features/php8.3
```
Comment (1RV34) [inline scripts/amd64.sh:520]: I'm not sure but shouldn't this be
```php
touch /home/vagrant/.homestead-features/php83
```
?
The `touch`es for the previous PHP versions were without `.`.

Comment (svpernova09) [reply scripts/amd64.sh:520]: Good catch, resolved in https://github.com/laravel/settler/commit/816304e1a8681118d8e4f9e46c62cfdd8f63d44c

**Pull request merged**